pipenv/project.py

```diff
@@ -237,11 +237,7 @@ def required_python_version(self):
 
     @property
     def project_directory(self):
-        if self.pipfile_location is not None:
-            return os.path.abspath(os.path.join(self.pipfile_location, os.pardir))
-
-        else:
-            return None
+        return os.path.abspath(os.path.join(self.pipfile_location, os.pardir))
```

Comment:
Return the project home even if there is no Pipfile existing.

*NOTE*: `$ pipenv --where` still prints an error if Pipfile doesn't exist.